README.md

```diff
@@ -29,9 +29,14 @@ Install the package `hub-xmlrpc-api`, available from SUSE Manager 4.1 and Uyuni
  - `HUB_API_URL`: URL to the Hub XMLRPC API endpoint
  - `HUB_CONNECT_TIMEOUT`: maximum number of seconds to wait for a response when connecting to a Server
  - `HUB_REQUEST_TIMEOUT`: maximum numbr of seconds to wait for a response when calling a Server method
+ - `USE_SSL`: use https protocol for communication between Daemon and peripheral servers.
 
  Default values should suffice in most settings.
 
+ ### Note
+ In order to make https work, in addition to setting `USE_SSL` flag to true, Certificates for all the servers will be needed on the machine where Daemon will be running. This can be achieved by copying  `RHN-ORG-TRUSTED-SSL-CERT` certificate file from each server's pub directory `https://<server-url>/pub/` to `/etc/pki/trust/anchors/` on the target machine where Dameon is running and then run the `update-ca-certificates`.
```

Comment:
```suggestion
In order to use https to connect to peripheral Servers, in addition to setting `USE_SSL` flag to true, SSL certificates for all the peripheral Servers need to be installed on the machine where the `hub-xmlrpc-api` service runs. This can be achieved by copying the `RHN-ORG-TRUSTED-SSL-CERT` certificate file from each peripheral Server's `pub` directory (`https://<server-url>/pub/`) to `/etc/pki/trust/anchors/` and then running the `update-ca-certificates` command.
```